documentation/book/ref-tls-sidecar.adoc

```diff
@@ -11,16 +11,31 @@ The TLS sidecar can be configured using the `tlsSidecar` property in:
 * `Kafka.spec.zookeeper`
 * `Kafka.spec.entityOperator`
 
-The TLS sidecar supports two additional options:
+The TLS sidecar supports three additional options:
 
 * `image`
 * `resources`
+* `logLevel`
 
 The `resources` property can be used to specify the xref:assembly-resource-limits-and-requests-{context}[memory and CPU resources] allocated for the TLS sidecar.
 
 The `image` property can be used to configure the container image which will be used.
 For more details about configuring custom container images, see xref:assembly-configuring-container-images-{context}[].
 
+The `logLevel` property can be used to specify the logging level which will be used.
```

Comment:
_Please change to:_

The `logLevel` property is used to specify the logging level.